docs/samples/Local TXDC Setup.md

```diff
@@ -64,10 +64,9 @@ helm install plato charts/tractusx-connector \
     --namespace cx \
     --create-namespace \
     --set fullnameOverride=plato \
-    --set controlplane.image.tag=0.2.0 \
+    --set controlplane.image.tag=latest \
```

Comment:
I tried the fixed command, but ran into the following error unfortunately.

```
Error: INSTALLATION FAILED: execution error at (tractusx-connector/templates/deployment-controlplane.yaml:191:54): .Values.postgresql.username is required
```

Considering the context that PostgreSQL is disabled in the preceding infrastructure setup, how about adding the following option?

```
--set controlplane.image.repository=edc-controlplane-memory-hashicorp-vault
```

In addition, even if specifying the option above, `postgresql.username`, `postgresql.password`, and `postgresql.jdbcUrl` still seem to be required, though any value is OK for them. According to the document, [there is a `postgresql.enabled` option](https://github.com/eclipse-tractusx/tractusx-edc/blob/57da0cb124a3efea54d7fe5059d2b33b816167bf/charts/tractusx-connector/README.md?plain=1#L216), but it doesn't seem to be used in any Helm chart.

```
$ git grep postgresql.enabled
.github/workflows/business-tests.yaml:              --set postgresql.enabled=true \
.github/workflows/business-tests.yaml:              --set postgresql.enabled=true \
charts/tractusx-connector-app/README.md:| runtime.postgresql.enabled | bool | `false` |  |
charts/tractusx-connector/README.md:| postgresql.enabled | bool | `false` |  |
docs/development/Run-business-tests-local.md:  --set postgresql.enabled=true \
docs/development/Run-business-tests-local.md:  --set postgresql.enabled=true \
```

Is this a documentation bug, and if so, should we fix it in this PR?